src/scripts/towns/BattleCafe.ts

```diff
@@ -169,85 +169,85 @@ class BattleCafeController {
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Star Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.at7Above10]] = new PokemonItem('Alcremie (Star Rainbow)', 0);
-
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]] = {};
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.dayClockwiseBelow5]] = new PokemonItem('Alcremie (Ribbon Vanilla)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.dayCounterclockwiseBelow5]] = new PokemonItem('Alcremie (Ribbon Ruby Cream)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.nightClockwiseBelow5]] = new PokemonItem('Alcremie (Ribbon Matcha)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.nightCounterclockwiseAbove5]] = new PokemonItem('Alcremie (Ribbon Mint)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.nightClockwiseAbove5]] = new PokemonItem('Alcremie (Ribbon Lemon)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.nightCounterclockwiseBelow5]] = new PokemonItem('Alcremie (Ribbon Salted)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.dayCounterclockwiseAbove5]] = new PokemonItem('Alcremie (Ribbon Ruby Swirl)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.dayClockwiseAbove5]] = new PokemonItem('Alcremie (Ribbon Caramel)', 0);
-        evolutions[GameConstants.AlcremieSweet[GameConstants.AlcremieSweet['Ribbon Sweet']]][GameConstants.AlcremieSpins[GameConstants.AlcremieSpins.at7Above10]] = new PokemonItem('Alcremie (Ribbon Rainbow)', 0);
+        evolutions[GameConstants.AlcremieSweet['Strawberry Sweet']] = {};
+        evolutions[GameConstants.AlcremieSweet['Strawberry Sweet']][GameConstants.AlcremieSpins.dayClockwiseBelow5] = new PokemonItem('Alcremie (Strawberry Vanilla)', 0);
+        evolutions[GameConstants.AlcremieSweet['Strawberry Sweet']][GameConstants.AlcremieSpins.dayCounterclockwiseBelow5] = new PokemonItem('Alcremie (Strawberry Ruby Cream)', 0);
+        evolutions[GameConstants.AlcremieSweet['Strawberry Sweet']][GameConstants.AlcremieSpins.nightClockwiseBelow5] = new PokemonItem('Alcremie (Strawberry Matcha)', 0);
```

Comment:
I was thinking something like this would be cleaner:
 
```ts
    private static evolutions: Record<GameConstants.AlcremieSweet, Record<GameConstants.AlcremieSpins, PokemonItem>> = {
        [GameConstants.AlcremieSweet['Strawberry Sweet']]: {
            [GameConstants.AlcremieSpins.dayClockwiseBelow5]: PokemonItem('Alcremie (Strawberry Vanilla)', 0),
            [GameConstants.AlcremieSpins.dayCounterClockwiseBelow5]: PokemonItem('Alcremie (Strawberry Ruby Cream)', 0),
            [GameConstants.AlcremieSpins.nightClockwiseBelow5]: PokemonItem('Alcremie (Strawberry Matcha)', 0),
            ... 
        },
        [GameConstants.AlcremieSweet['Love Sweet']: {
            ...
        },
        ...
    }
```

Reply:
Thanks. I tried to do this, but i had no idea how the type should look... But it works! I have submitted it